packages/snaps-rpc-methods/src/permitted/scheduleBackgroundEvent.ts

```diff
@@ -78,24 +73,18 @@ export type ScheduleBackgroundEventParameters = InferMatching<
 >;
 
 /**
- * Generates a `DateTime` object based on if a duration or date is provided.
+ * Get the schedule for a background event based on the provided parameters.
  *
- * @param params - The validated params from the `snap_scheduleBackgroundEvent` call.
- * @returns A `DateTime` object.
+ * @param params - The parameters for the background event.
+ * @returns The schedule parameters for the background event.
  */
-function getStartDate(params: ScheduleBackgroundEventParams) {
```

Comment:
The logic for scheduling was previously done in `snaps-rpc-methods`, but handling this from the cronjob controller is easier to reduce duplication.